extensions/wikia/WikiFeatures/templates/WikiFeaturesSpecial_feature.php

```diff
@@ -1,39 +1,39 @@
-<li class="feature" data-name="<?= $feature['name'] ?>" data-heading="<?= wfMessage('wikifeatures-feature-heading-'.$feature['name'])->plain() ?>">
+<li class="feature" data-name="<?= $feature['name'] ?>" data-heading="<?= wfMessage( 'wikifeatures-feature-heading-' . $feature['name'] )->escaped(); ?>">
 	<h3>
-		<?= wfMsg('wikifeatures-feature-heading-'.$feature['name']) ?>
-		<? if (isset($feature['active'])): ?>
+		<?= wfMessage( 'wikifeatures-feature-heading-' . $feature['name'] )->escaped(); ?>
+		<? if ( isset( $feature['active'] ) ): ?>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "active" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)